packages/esm-patient-test-results-app/src/grouped-timeline/grouped-timeline.types.ts

```diff
@@ -0,0 +1,43 @@
+import { OBSERVATION_INTERPRETATION } from '@openmrs/esm-patient-common-lib';
```

Comment:
Pedantic but shouldn't this be `import type`?

Reply:
@ibacher thanks! -> Today I Learned...